manifest_staging/charts/csi-secrets-store-provider-azure/templates/arc-monitoring.yaml

```diff
@@ -183,10 +183,10 @@ spec:
           resources:
             requests:
               cpu: 50m
-              memory: 100Mi
+              memory: 250Mi
```

Comment:
Have these limits been tested? If yes, could you update the PR description on why this change is being made and a note about the testing?

Reply:
done